RealmMapView/RealmMapView.swift

```diff
@@ -249,6 +259,27 @@ public class RealmMapView: MKMapView {
         rlmConfiguration.schemaVersion = configuration.schemaVersion
         return rlmConfiguration
     }
+  
+    public func config(entityName : String!, latitudeKeyPath : String!, longitudeKeyPath : String!, titleKeyPath : String!, subtitleKeyPath : String!) {
+        self.entityName = entityName
+        self.latitudeKeyPath = latitudeKeyPath
+        self.longitudeKeyPath = longitudeKeyPath
+        self.titleKeyPath = titleKeyPath
+        self.subtitleKeyPath = subtitleKeyPath
+    }
+  
+    private func assertConfig() -> Bool {
```

Comment:
Please rebase off of master from my most recent commit. I made `titleKeyPath` and `subtitleKeypath` optional now (if nil then they are simply not shown on the callout). You can then adjust the assert to only check for `entityName`, `latitudeKeyPath` and `longitudeKeyPath`.